proposal/oscar97123/proposal.rst

```diff
@@ -88,21 +99,27 @@ both.
 Engineering infrastructure
 ==========================
 
-<TO-DO>
+There will be 2 branches in the development. The first branch(master) holds the stable version of the software(with unit test assurance).
```

Comment:
I like the explicit description of the branches.  But please note that in your repository, you used `main` instead of `master`.